playground/tools/scrap.py

```
@@ -0,0 +1,357 @@
+    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/125.0.0.0 Safari/537.36",
+    "Accept": "application/json",
+    "Accept-Language": "ko-KR,ko;q=0.9",
+    "Connection": "keep-alive",
+    "Content-Type": "application/json",
+})
+
+# =============================================================================
+# 🛠️ 1. 기본 상품 검색 도구
+# =============================================================================
+@tool
+def search_basic_products(keyword: str, gender: str = "A", minPrice: int = 0, maxPrice: int = 999999, color: str = "",shoeSize: str = 0)-> list:
```

Ruff (0.12.2)

Too many arguments in function definition (6 > 5)

(PLR0913)